app/controllers/users.js

```diff
@@ -29,4 +30,16 @@ const usersList = async (req, res, next) => {
   }
 };
 
-module.exports = { signUp, signIn, usersList };
+const admin = async (req, res, next) => {
+  if (req.user.role === 'user') {
```

Comment:
Maybe directly check that the user is `admin`, because maybe in the future you would haver more than two user roles